REFERENCES.md

```diff
@@ -0,0 +1,183 @@
+# Scientific Articles
+We encourage you to also check out work by the group behind 
+GluonTS. They are grouped according to topic and ordered 
+chronographically.
+
+### Methods
+A number of the below methods are available in GluonTS.
+
+```
+@inproceedings{salinas2019high,
+	Author = {Salinas, David and Bohlke-Schneider, Michael and Callot, Laurent and Gasthaus, Jan},
+	Booktitle = {Advances in Neural Information Processing Systems},
+	Title = {High-Dimensional Multivariate Forecasting with Low-Rank Gaussian Copula Processes},
+	Year = {2019}
+}
+```
+
+Deep Factor models, a global-local forecasting method.
+```
+@inproceedings{wang2019deepfactors,
+	Author = {Wang, Yuyang and Smola, Alex and Maddix, Danielle and Gasthaus, Jan and Foster, Dean and Januschowski, Tim},
+	Booktitle = {International Conference on Machine Learning},
+	Pages = {6607--6617},
+	Title = {Deep factors for forecasting},
+	Year = {2019}
+}
+```
+DeepAR, an RNN-based probabilistic forecasting model.
+```
+@article{flunkert2019deepar,
+	Author = {Salinas, David and Flunkert, Valentin and Gasthaus, Jan and Tim Januschowski},
+	Journal = {International Journal of Forecasting},
+	Title = {DeepAR: Probabilistic forecasting with autoregressive recurrent networks},
+	Year = {2019}
+}
+```
+A flexible way to model probabilistic forecasts via spline quantile forecasts.
+```
+@inproceedings{gasthaus2019probabilistic,
+	Author = {Gasthaus, Jan and Benidis, Konstantinos and Wang, Yuyang and Rangapuram, Syama Sundar and Salinas, David and Flunkert, Valentin and Januschowski, Tim},
+	Booktitle = {The 22nd International Conference on Artificial Intelligence and Statistics},
+	Pages = {1901--1910},
+	Title = {Probabilistic Forecasting with Spline Quantile Function RNNs},
+	Year = {2019}
+}
+```
+Using RNNs to parametrize State Space Models.
+```
+@inproceedings{rangapuram2018deep,
+	Author = {Rangapuram, Syama Sundar and Seeger, Matthias W and Gasthaus, Jan and Stella, Lorenzo and Wang, Yuyang and Januschowski, Tim},
+	Booktitle = {Advances in Neural Information Processing Systems},
+	Pages = {7785--7794},
+	Title = {Deep state space models for time series forecasting},
+	Year = {2018}
+}
+```
+A scalable state space model.
+```
+@inproceedings{seeger2016bayesian,
```

Comment:
This one is not available, can you remove it from the list?

It would be really nice to have a link to the source code.

Reply:
How about I leave the paper in here, but with a remark that this isn't part of GluonTS currently?

Reply:
I think it is better to add pointers only to papers that are in Gluon-ts otherwise it is arguable why putting this reference rather than another one but I let you take the final decision ;)